remediation_worker/jobs/ec2_close_port_1433/ec2_close_port_1433.py

```diff
@@ -56,8 +56,207 @@ def parse(self, payload):
+                                'SecurityGroupRuleId': rule["SecurityGroupRuleId"],
+                                'SecurityGroupRule': {
+                                    'IpProtocol': 'tcp',
+                                    'FromPort': port + 1,
+                                    'ToPort': rule["ToPort"],
+                                    'CidrIpv6': '::/0'
+                                }
+                            }
+                        ]
+                    )
+                elif rule["FromPort"] < port and rule["ToPort"] > port:
+                    logging.info("    executing client.modify_security_group_rules")
+                    logging.info('      CidrIpv6="::/0"')
+                    logging.info(f"      FromPort={rule['FromPort']}")
+                    logging.info(f"      GroupId={security_group_id}")
+                    logging.info('      IpProtocol="tcp"')
+                    logging.info(f"      ToPort={port - 1}")
+                    client.modify_security_group_rules(
+                        GroupId=security_group_id,
+                        SecurityGroupRules=[
+                            {
+                                'SecurityGroupRuleId': rule["SecurityGroupRuleId"],
+                                'SecurityGroupRule': {
+                                    'IpProtocol': 'tcp',
+                                    'FromPort': rule["FromPort"],
+                                    'ToPort': port - 1,
+                                    'CidrIpv6': '::/0'
+                                }
+                            }
+                        ]
+                    )
+
+                    logging.info("    executing client.authorize_security_group_ingress")
+                    logging.info('      CidrIpv6="::/0"')
+                    logging.info(f"      FromPort={port + 1}")
+                    logging.info(f"      GroupId={security_group_id}")
+                    logging.info('      IpProtocol="tcp"')
+                    logging.info(f"      ToPort={rule['ToPort']}")
+                    client.authorize_security_group_ingress(
+                        GroupId=security_group_id,
+                        IpPermissions=[
+                            {
+                                'FromPort': port + 1,
+                                'IpProtocol': 'tcp',
+                                'Ipv6Ranges': [
+                                    {
+                                        'CidrIpv6': '::/0',
+                                    },
+                                ],
+                                'ToPort': rule["ToPort"]
+                            },
+                        ]
+                    )
+
     def remediate(self, client, instance_id):
-        """Block public access to port 1433 of all security groups attached to an EC2 instance.
+        """Block public access to port 22 of all security groups attached to an EC2 instance.
```

Comment:
Is this a cut & paste error as the file is ec2_close_port_1433, but comment mentions port 22 thru out the diff?